create.go

```diff
@@ -52,7 +52,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			}
 
 			patches, err := r.computeCreateEventPatches(ctx, newObj)
-			if err != nil {
+			if tenant.IsAPINotAvailable(err) {
+				r.logger.LogCtx(ctx, "level", "debug", "message", "tenant cluster is not available")
```

Comment:
Since we already use two places for tenant package, I believe this is a better approach and creating a debug message like this is more clear to the user.